### PR TITLE
fix(ci): run on ubuntu 22.04

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

**Comment by lucasfernog** [comment]
https://github.com/tauri-apps/tauri/actions/runs/11326945077/job/31496943167#step:10:748

**Review by amrbashir — DISMISSED** [review]

**Review by amrbashir — APPROVED** [review]

File: .docker/cross/aarch64.Dockerfile [hunk 1]
```diff
@@ -41,4 +41,4 @@ ENV CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc \
 
 RUN dpkg --add-architecture arm64
 RUN apt-get update
-RUN apt-get install --assume-yes --no-install-recommends libssl-dev:arm64 libdbus-1-dev:arm64 libsoup2.4-dev:arm64 libssl-dev:arm64 libgtk-3-dev:arm64 webkit2gtk-4.0-dev:arm64 libappindicator3-1:arm64 librsvg2-dev:arm64 patchelf:arm64
+RUN apt-get install --assume-yes --no-install-recommends libssl-dev:arm64 libdbus-1-dev:arm64: libsoup2.4-devarm64 libssl-dev:arm64 libgtk-3-dev:arm64 webkit2gtk-4.0-dev:arm64 libappindicator3-1:arm64 librsvg2-dev:arm64 patchelf:arm64
```
Comment (FabianLars) [inline .docker/cross/aarch64.Dockerfile:44]: that doesn't look right

Comment (lucasfernog) [reply .docker/cross/aarch64.Dockerfile:44]: aaaaaaaaaaa wrong find & replace now please get out of here 🤣

Comment (FabianLars) [reply .docker/cross/aarch64.Dockerfile:44]: i was never here 🏃‍♂️

**Pull request merged**